base/applications/mspaint/winproc.cpp

```diff
@@ -657,6 +672,20 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             if (pd.hDevNames)
                 GlobalFree(pd.hDevNames);
             break;
+        case IDM_FILESEND:
+            if (PathFileExists(g_szFileName))
+            {
+                CStringW strSubject;
+                strSubject.LoadString(IDS_MAILSUBJECT);
+
+                // NOTE: "%22" is for a double quote
+                CString strCmdLine;
```

Comment:
Why not using directly unicode strings here `CStringW` instead of the TCHAR thing?